simapp/state.go

```diff
@@ -155,8 +162,19 @@ func AppStateRandomizedFn(
 	simManager *module.SimulationManager, r *rand.Rand, cdc codec.JSONCodec,
 	accs []simtypes.Account, genesisTimestamp time.Time, appParams simtypes.AppParams,
 ) (json.RawMessage, []simtypes.Account) {
-	numAccs := int64(len(accs))
 	genesisState := NewDefaultGenesisState(cdc)
+	return AppStateRandomizedFnWithState(simManager, r, cdc, accs, genesisTimestamp, appParams, genesisState)
+}
+
+// AppStateRandomizedFnWithState creates calls each module's GenesisState generator function
+// and creates the simulation params
+// genesisState is the genesis state of the app.
+func AppStateRandomizedFnWithState(
```

Comment:
Can you add a comment that this will not exist in v0.47, but be replaced by `AppStateRandomizedFn` with an extra argument?